### ISSUE TITLE
Request for loss curve

### ISSUE THREAD

**Issue by ssarkar2** [rafaelsf80/vertex-flant5xxl-multitask-finetuning#1]
Hi @rafaelsf80 ,
Would it be possible to provide the training loss curve for this finetuning task.

Thanks
